third_party/move/move-model/bytecode/src/function_target_pipeline.rs

```diff
@@ -270,24 +270,31 @@ impl FunctionTargetsHolder {
         self.targets.entry(*id).or_default().insert(variant, data);
     }
 
-    /// Processes the function target data for given function.
+    /// Return true if this transformed code, or false otherwise
```

Comment:
Please bring back the deleted sentence, than can add 'Returns true if`.

Reply:
fixed